open-api/rest-catalog-open-api.yaml

```diff
@@ -770,6 +770,30 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/namespaces/{namespace}/tables/{table}/metrics/scan:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+      - $ref: '#/components/parameters/namespace'
+      - $ref: '#/components/parameters/table'
+
+    post:
+      tags:
+        - Catalog API
+      summary: Send a scan report to this endpoint to be processed by the backend
+      description:
+        Send a scan report to this endpoint to be processed by the backend
+      operationId: sendScanReport
+      requestBody:
+        description: The request containing the scan report to be sent
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ScanReportRequest'
+        required: true
+      responses:
+        200:
```

Comment:
We probably want to include the 4XX and 5XX codes here as well like other endpoints have, just for completeness.

Reply:
I had those initially defined (at least the 4xx), but Ryan suggested to remove those (see https://github.com/apache/iceberg/pull/5407#discussion_r947180948)

Reply:
I think we still need the 5xx and other responses. I just meant that it is debatable whether the metrics endpoint should ever return a failure that the user was not authenticated. It's unlikely that the caller would do anything about it, right?

Reply:
Should this be a 204? Or do we expect to return something?

Reply:
changed this to 204 and added the other 4xx/5xx error codes as well